src/components/CityDetails/CityDetails.js

```diff
@@ -27,25 +28,20 @@ class CityDetails extends Component {
   render() {
     return (
       <section className="city-details">
-        {this.state.error && <h2>{this.state.error}</h2>}
-        {this.state.city &&
```

Comment:
I think we still need this chunk of code!

Reply:
Def! Nice catch. Just added it back in so it should be good to go on this PR.

Reply:
I believe you fixed it while I was reviewing the code...